packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -248,4 +248,6 @@ export interface FeatureToggles {
   unifiedHistory?: boolean;
   lokiLabelNamesQueryApi?: boolean;
   investigationsBackend?: boolean;
+  folderCounts?: boolean;
```

Comment:
Should these names be this generic? I think something like `k8sFolderCounts` would read better, wdyt?

Reply:
changing it :)